### PR TITLE
feat(gandalf/frontend): added variables.css

### PR DESCRIPTION
**Pull request by dimonl**
added variables.

### CODE REVIEW

File: frontend/src/app/libs/common-components/shared/variables.scss [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+$color-black-3: rgba($color-black, .03);
+// whites
+$color-white-100: rgba($color-white);
+$color-white-75: rgba($color-white, .75);
+$color-white-50: rgba($color-white, .50);
+$color-white-30: rgba($color-white, .30);
+$color-white-10: rgba($color-white, .10);
+$color-white-5: rgba($color-white, .05);
+// typography
+$font-main: 'Roboto Bold';
```
Comment (clain-atomi) [inline frontend/src/app/libs/common-components/shared/variables.scss:37]: lets rename it to `font-main-bold` 
also add missing spaces before `'` check all styles

File: frontend/src/app/libs/common-components/shared/variables.scss [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+$font-main-regular:'Roboto Regular';
+$font-main-medium:'Roboto Medium';
+// typography styles
+.text-h1 {
```
Comment (clain-atomi) [inline frontend/src/app/libs/common-components/shared/variables.scss:41]: let's make all these styles `@mixin`

File: frontend/src/app/libs/common-components/shared/variables.scss [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    letter-spacing: 0;
+    line-height: 16px;
+}
+
+.text-small-bold {
+    font-family: $font-main;
+    font-size: 12px;
+    letter-spacing: 0;
+    line-height: 16px;
+}
```
Comment (clain-atomi) [inline frontend/src/app/libs/common-components/shared/variables.scss:95]: add new line at the and of file

**Review by clain-atomi — CHANGES_REQUESTED** [review]

File: frontend/src/app/libs/common-components/shared/variables.scss (hunk 1 above)
Comment (dimonl) [reply frontend/src/app/libs/common-components/shared/variables.scss:37]: done

File: frontend/src/app/libs/common-components/shared/variables.scss (hunk 2 above)
Comment (dimonl) [reply frontend/src/app/libs/common-components/shared/variables.scss:41]: mixin added

File: frontend/src/app/libs/common-components/shared/variables.scss (hunk 3 above)
Comment (dimonl) [reply frontend/src/app/libs/common-components/shared/variables.scss]: added

File: frontend/src/app/libs/common-components/shared/variables.scss [hunk 4]
```diff
@@ -0,0 +1,78 @@
+@mixin text-view($font-family, $font-size, $letter-spacing, $line-height) {
```
Comment (clain-atomi) [inline frontend/src/app/libs/common-components/shared/variables.scss:1]: not exactly, i mean make all .class mixins, co consumer will be able to include such mixin to it's own classes

**Review by clain-atomi — CHANGES_REQUESTED** [review]

File: frontend/src/app/libs/common-components/shared/variables.scss (hunk 4 above)
Comment (dimonl) [reply frontend/src/app/libs/common-components/shared/variables.scss]: sorry, changed

**Pull request merged**